test/parser.test.js

```diff
@@ -1,7 +1,10 @@
 const test = require('ava')
+const nock = require('nock')
+const path = require('path')
 const toArray = require('stream-to-array')
 
 const {xlsxParser} = require('../lib/parser/xlsx')
+const {csvParser} = require('../lib/parser/csv')
```

Comment:
this should be combined with https://github.com/datahq/data.js/pull/44/files#diff-7bf59ee18e32a02823031db5b25b384cR9 :

`const {csvParser , guessParseOptions} = require('../lib/parser/csv')`